README.md

```diff
@@ -83,6 +83,9 @@ code name
 # 0x14 formerly had the name "sha3", now deprecated
 ```
 
+## Compatibility note
+
+Implementations should make sure keep the string code `sha3` as an alias for `sha3-512` so that `encode('sha3', digest) == encode('sha3-512', digest)`.
```

Comment:
This multihash repo defines a spec, not an API, so the proposed wording that references a "string code" feels weird to me (many implementations will not use a string to specify the hash type).  What about something like this instead:

> Implementations should be aware that a previous version of this spec used the label `SHA3` for the SHA3-512 hash (code `0x14`).  If your implementation allows hash types to be specified as a string, you should consider making `SHA3` and `SHA3-512` aliases, or disallow the label `SHA3` entirely.